### PR TITLE
Badge Component Story

### PR DESCRIPTION
**Pull request by nathanstilwell**
storybook-badge

### CODE REVIEW

File: packages/storybook-ui-components/stories/1-Badge.stories.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  components: Badge,
+  decorators: [withKnobs],
+};
+
+export const Demo = () => (
+  <Badge
+    content={text("Content Prop", "badge")}
+    intent={select(
+      "Intent",
+      m(["neutral", "success", "warning", "danger"]),
```
Comment (nathanstilwell) [inline packages/storybook-ui-components/stories/1-Badge.stories.js]: The `m` is temporary. That was just me being lazy. This should be a name that increases the readability here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+WIP
+
+## Clone the repo
+
+```
+> git clone https://github.com/cockroachdb/ui/
+```
+
+The UI repo is a monorepo managed by [Lerna](https://lerna.js.org/). Once you clone the repo, be sure to [bootstrap](). _In the following examples, I'm using `npx`. If you do this very often you may consider installing Lerna globally (`npm i -g lerna`)._
```
Comment (j-low) [inline CONTRIBUTING.md]: nit: looks like there is a link missing at `[bootstrap]()`

File: packages/storybook-ui-components/stories/0-Welcome.stories.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+export default {
+  title: "Welcome",
+  component: Welcome,
+};
+
+export const ToStorybook = () => <Welcome showApp={linkTo("Button")} />;
+
+ToStorybook.story = {
+  name: "blah blah Storybook",
```
Comment (j-low) [inline packages/storybook-ui-components/stories/0-Welcome.stories.js]: Flagging placeholder text

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+## Branch and develop
+
+Create a branch for your work. When you are ready, create a [pull-request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests) for your work.
+
+```
+> git checkout -b package-name--branch-name-describing-work
+```
+
+Branch names should contain the name of the package you are working on as well as a description of the work you are doing.
```
Comment (laurenbarker) [inline CONTRIBUTING.md]: If we are using forks then the name of the branch doesn't really matter.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+The UI repo is a monorepo managed by [Lerna](https://lerna.js.org/). Once you clone the repo, be sure to [bootstrap](). _In the following examples, I'm using `npx`. If you do this very often you may consider installing Lerna globally (`npm i -g lerna`)._
+
+```
+> cd path/to/ui
+> npx lerna bootstrap
+```
+
+## Branch and develop
```
Comment (laurenbarker) [inline CONTRIBUTING.md]: I think we should follow the same pattern we use with all of our other apps (i.e., fork --> branch --> PR). Is that what this guide is describing?

The "Where to push" section of the [CRDB contributing guidelines](https://wiki.crdb.io/wiki/spaces/CRDB/pages/181633464/Your+first+CockroachDB+PR) has a description of the CRDB process.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+Create a branch for your work. When you are ready, create a [pull-request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests) for your work.
+
+```
+> git checkout -b package-name--branch-name-describing-work
+```
+
+Branch names should contain the name of the package you are working on as well as a description of the work you are doing.
+
+## Publishing
```
Comment (laurenbarker) [inline CONTRIBUTING.md]: Thanks for adding these instructions! 💯

File: packages/storybook-ui-components/stories/1-Badge.stories.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { withKnobs, text, select } from "@storybook/addon-knobs";
+import { Badge } from "@cockroachlabs/ui-components";
```
Comment (laurenbarker) [inline packages/storybook-ui-components/stories/1-Badge.stories.js]: Ah, I see! So we will import components from other packages and display them in one storybook instance?

File: CONTRIBUTING.md (hunk 2 above)
Comment (nathanstilwell) [reply CONTRIBUTING.md]: good catch.

File: packages/storybook-ui-components/stories/0-Welcome.stories.js (hunk 3 above)
Comment (nathanstilwell) [reply packages/storybook-ui-components/stories/0-Welcome.stories.js]: 😆 👍

File: CONTRIBUTING.md (hunk 4 above)
Comment (nathanstilwell) [reply CONTRIBUTING.md]: Nice catch. I will change this section to propose _fork and pull_ over _branch and pull_.

File: packages/storybook-ui-components/stories/1-Badge.stories.js (hunk 7 above)
Comment (nathanstilwell) [reply packages/storybook-ui-components/stories/1-Badge.stories.js]: Yes, that was my plan. To have the _Storybook_ be a separate package that will depend on UI Components to keep a separation of concerns.

**Comment by nathanstilwell** [comment]
@j-low @laurenbarker Just a note to say that updates to `CONTRIBUTING.md` will be made in a separate PR.

**Pull request merged**